packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -577,7 +577,7 @@ ProductList.getProps = async ({res, params, location, api}) => {
 
     // Set the `cache-control` header values to align with the Commerce API settings.
     if (res) {
-        res.set('Cache-Control', 'public, must-revalidate, max-age=900')
+        res.set('Cache-Control', 'max-age=900')
```

Comment:
According to John, having both max-age and must-revalidate is an anti-pattern: https://grayduck.mn/2021/09/13/cache-control-recommendations/